src/main/kotlin/com/faforever/userservice/controller/OAuthController.kt

```diff
@@ -134,4 +139,18 @@ class OAuthController(
         }.flatMap { redirectUrl ->
             redirect(response, redirectUrl)
         }
+
+    @PostMapping("revokeTokens")
```

Comment:
Do we not need to expose this endpoint in the SecurityConfig?

Reply:
no it is covered by /**